firmware/integration/rusefi_config.txt

```diff
@@ -1111,7 +1111,9 @@ int16_t tps2Max;Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\
 	brain_pin_e can2TxPin;set_can2_tx_pin X
 	brain_pin_e can2RxPin;set_can2_rx_pin X
 	uint8_t unused1108;;"units", 1, 0, -20, 100, 0
-	uint32_t[3] unused_former_warmup_target_afr;;"units", 1, 0, -20, 100, 0
+	bit[5] unused1116
```

Comment:
"bit[5]" i am not sure if that's even supported

please use X number of uint8_t